include/mra/tasks/compress.h

```diff
@@ -132,7 +133,7 @@ namespace mra
           auto rcoeffs_view = d.current_view();
           auto hgT_view = hgT.current_view();
 
-          submit_compress_kernel(key, N, K, coeffs_view, rcoeffs_view, hgT_view,
+          submit_compress_kernel(key, N, K, is_ns, coeffs_view, rcoeffs_view, hgT_view,
```

Comment:
We need to set whether the result is in non-standard form.

Reply:
Fixed in a recent commit